static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -51,15 +27,14 @@ export default function GroupingInfo({
   showGroupingConfig,
   group,
 }: GroupingSummaryProps) {
-  const [configOverride, setConfigOverride] = useState<string | null>(null);
   const hasStreamlinedUI = useHasStreamlinedUI();
 
   const {groupInfo, isPending, isError, isSuccess, hasPerformanceGrouping} =
     useEventGroupingInfo({
       event,
       group,
       projectSlug,
-      query: configOverride ? {config: configOverride} : {},
+      query: {},
```

Comment:
Does the `query` param still need to exist in `useEventGroupingInfo`? If so, could we rely on a default there rather than passing an empty object here?